queries/gsuite_queries/GSuite_Many_Docs_Downloaded_Query.yml

```diff
@@ -0,0 +1,18 @@
+AnalysisType: scheduled_query
+Description: Query to search high document download counts by users.
+Enabled: true
+Query: |
+    SELECT
+        actor:email AS user,
+        ARRAY_AGG( DISTINCT parameters:doc_title) AS downloaded_files,
+        ARRAY_SIZE(downloaded_files) as download_count
+    FROM gsuite_activityevent
+    WHERE p_occurs_since('1 day')
+        AND name = 'download'
+    GROUP BY actor:email
+    HAVING download_count > 10
```

Comment:
I wonder if we should time_slice based on the hour

Reply:
added time_slicing